eq-author-api/schema/tests/comments.test.js

```diff
@@ -73,6 +73,34 @@ describe("comments", () => {
     });
   });
 
+  it("should add multiple comments on question page and then query those comments", async () => {
+    await createComment(ctx, {
+      pageId: createdQuestionPage.id,
+      commentText: "a new comment is created",
+    });
+
+    await createComment(ctx, {
+      pageId: createdQuestionPage.id,
+      commentText: "a 2nd comment is created",
+    });
+
+    const queryNewComment = await queryComments(ctx, {
```

Comment:
change this const to `queryNewComments` as it is is multiple of them.

Reply:
makes sense - ta